qa/tasks/cephfs/test_volume_client.py

```diff
@@ -970,6 +970,26 @@ def test_get_object(self):
             obj_data = obj_data
         )))
 
+    def test_put_object_versioned(self):
+        vc_mount = self.mounts[1]
+        vc_mount.umount_wait()
+        self._configure_vc_auth(vc_mount, "manila")
+
+        obj_data = 'test_data'
+        obj_name = 'test_vc_ob_2'
+        pool_name = self.fs.get_data_pool_names()[0]
+        self.fs.rados(['put', obj_name, '-'], pool=pool_name, stdin_data=obj_data)
+
+        with self.assertRaises(CommandFailedError):
```

Comment:
I haven't tested this method (since I hit some issue for running tests on my machine) to know if this works as expected. Will update as soon as I test it.

Reply:
I forgot to mention yesterday, I've ran the test and this works fine.